src/components/List/List.js

```diff
@@ -1,3 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import './list.css'
 
+export const List = ({isOrdered, listItems, ...props}) => {
+
+    if (isOrdered) {
+      return (
+        <ol>
+        {listItems.map((listItem, i) => {
+          return (
+            <li className={`storybook-li`}>{listItem}</li>
```

Comment:
Please include a key for your React component output: You can reference here: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Does this work for a key? 
`<li key={i} className={`storybook-li`}>{listItem}</li>`